oe_webtools_analytics/tests/src/Kernel/WebtoolsAnalyticsSearchServiceTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_analytics\Kernel;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\oe_webtools_analytics\Entity\WebtoolsAnalyticsSearch;
+
+/**
+ * Test to ensure 'WebtoolsAnalyticsSearch' service is reachable.
```

Comment:
I would remove this test, WebtoolsAnalyticsSearch shouldn't be a service (it is a value-object class for the event), and testing that it exists doesn't add much.